core/domain/classroom_config_domain.py

```diff
@@ -429,3 +429,79 @@ def to_dict(self) -> ImageDataDict:
             'bg_color': self.bg_color,
             'size_in_bytes': self.size_in_bytes,
         }
+
+
+class ClassroomIdToIndexDict(TypedDict):
+    """Dict type for ClassroomIdToIndex object."""
+
+    classroom_id: str
+    classroom_index: int
+
+
+class ClassroomIdToIndex:
```

Comment:
Why not just a field, like you described in https://docs.google.com/document/d/1wkLG9W6vIX3AeEnQCnKjfcwh7xgBk5-maNznRF8cgvQ/edit#heading=h.dizjk8vfjk0 ?

This change seems a bit too excessive

Reply:
Ah I picked the wrong option and implemented it. I will open a new PR today. my bad.